src/Application/Regulation/Command/SaveRegulationGeneralInfoCommand.php

```diff
@@ -22,6 +22,7 @@ final class SaveRegulationGeneralInfoCommand implements CommandInterface
     public ?\DateTimeInterface $endDate = null;
     public array $additionalVisas = [];
     public array $additionalReasons = [];
+    public ?string $visaModel = null;
```

Comment:
(Détail) Indiquer dans le nom de l'attribut qu'on stocke l'uuid et pas l'entité en entier ? Ça peut éviter des erreurs

```suggestion
    public ?string $visaModelUuid = null;
```